### DISCUSSION TITLE
Dataset für A-Z

### DISCUSSION THREAD

**Discussion by ghost** [Estheryu991/SwissGerman_Dictionary#54] [General]
A:
aabändle: ig möcht mit dir aabändle
Versuch eine beziehung anzuknüpfen

aabränte: Ig ha ds Brot la aabrönne
Anbrennen

aabruche: Mir hei z Müesli aabrucht
Zu benutzen beginnen

aachätsche: Si het z Sushi aagchätscht
Annagen

aacho: Der Zug isch aacho
Ankommen

Aafa: Er het aagfange
Anfangen

aafüüre: Der ETH isch aagfüüret worde
Anzünden

aagä: Si het d Üebige aagä
Angeben

aagüfele: Ds Biud isch ar Wand aagüfelet
Anstecken, anheften

aahou: Si het am liebschte der Aahou
Anschnitt (Brot)

aalegi: Kleider

Ääli: Ds Paar macht viu Ääli. (?)
Liebkosung

aaschnäggle:

**Comment by andrinr** [comment]
aahöilig: de afang und s endi vom brot

**Comment by esthicodes** [comment]
- aalüte: anrufen
Ich lüte dich an.
 
- aaschnäggle: negativ
- aba! : ach was!

- abetroole: hinunterfallen
Die Kind isch abetroolet
Die Kind ist hinuntergefallen.

**Comment by esthicodes** [comment]
i ga allpot uf Züri

ich gehe dauernd nach Zürich

- i ha efang emau es billet kouft, weiss aber nonid ob i ga

ich habe vorerst ein ticket gekauft, weiss aber noch nicht ob ich hingehe

- afe, efang: vorerst, vorläufig, schon

- Mein Nacken ist kalt darum habe ich einen Schal angezogen

Mi äcke isch kalt, drum hani e Schaal agleit

- Das isch mer abverheit
Das ist mir mislungen

- Mir sind abzottlet
Wir sind weggegangen.

**Comment by esthicodes** [comment]
i reise albe mitem zug, wenn i verreise

ich verreise jeweils mit dem zug

https://people.math.ethz.ch/~pink/ftp/Lineare-Algebra-Zusammenfassung-20161006.

**Comment by esthicodes** [comment]
- Chirsi: Kirschen
chyschterig: heiser

- chlädere: klettern 
https://people.math.ethz.ch/~pink/ftp/Lineare-Algebra-Zusammenfassung-20161006.pdf klettere gerne 

Ech gah regelmässig ga chlättere

- chläpfe: ohrfeigen

- Chlapf: Knall, Ohrfeige, Auto

- chlefele: klappern, klirren
(Bern-Luzern)
Öppis chlefelt am Outo.


- chly: klein


- chlööne: klagen, jammern, quengeln

hör uf chlööne!

- Chlüder: nomen 
Moneten, Geld

- Chlütter
du hesch z'vöu Chlütter

- Chlupf: Erschrecken, plötzlicher schreck

![image](https://user-images.githubusercontent.com/78131082/200139241-52f5256f-96e2-4ae3-accc-087c55fee8f1.jpeg)


- Chniepe: immer jammern und klagen

- Chnode: Gelenkknoten 

- Chnöi, Chnüü: Knie

- chnöilige: kniend

- Chnöpfli: Mehlspeise, ähnlich Wie Spätzle? 
Morn choche mer Spätzli.

- chnorze: sich abmühen, abracken
Du bisch so verchnorzt.

- chnütsche: quetschen, pressen 

- chnuuschte: kneten, drücken

- chöcherle: leise kochen 🧑‍🍳 
Mer mus das langsam la chöcherle.

- choder: Spucke, Halsschleim

**Comment by esthicodes** [comment]
Chöli: Grünkohl
- [x] choldere: trotzen
> Isch trotze 

- [x] Choli: schwarzes Pferd, Rappe
>ich reite auf meine Choli.

- [ ] Chöözete: Sudelei
> Ich chöözete immer beim essen.

- [ ] Chöpfler: Kopfsprung
> Chöpfler

- [ ]  chöpflige: Kopf voran
> Ich renne chöpflige durch die Wand.

- [ ] chosle: mit Wasser oder nasser Erde spielen 
> 

- [x] chotze: erbrechen
- [ ] chöie: kauen

- [ ] Chräbeli: Anisgebäck zu Weilnachten 
> Ich esse ein Chräbeli. 

- [ ] chräbele: kraulen, liebkosen 
> Mein Hund mag es wenn ich ihn chräbele.

**Comment by ghost** [comment]
Some Links for Studying Swiss German 
[`Chuchichäschtli, Chäschüechli etc...`](https://www.eldrid.ch/swgerman.htm)

**Reply by ghost** [reply]
[`C Programming Language 2nd Edition by Brian W. Kernighan & Dennis M. Ritchie Solutions`](https://github.com/hiverkiya/Dennis-Ritchie-C-Solutions)

Diese zweite Auflage beschreibt C, wie es durch den ANSI-Standard definiert ist. Dieses Buch soll dem Leser helfen, die Programmierung in C zu erlernen. Das Buch setzt eine gewisse Vertrautheit mit grundlegenden Programmierkonzepten wie Variablen, Zuweisungsanweisungen, Schleifen und Funktionen voraus. Ein unerfahrener Programmierer sollte in der Lage sein, das Buch zu lesen und sich die Sprache anzueignen.
Merkmale
Alle Beispiele sind getestet und in maschinenlesbarer Form.

Es werden verschiedene Aspekte von C näher erläutert, wobei der Schwerpunkt auf Beispielen vollständiger Programme und nicht auf isolierten Fragmenten liegt.

Es behandelt grundlegende Datentypen, Operatoren und Ausdrücke.

Behandelt werden Funktionen und Programmstruktur, externe Variablen, Geltungsbereichsregeln und mehrere Quelldateien, wobei auch auf den Präprozessor eingegangen wird.

Außerdem wird eine Schnittstelle zwischen C-Programmen und dem UNIX-Betriebssystem beschrieben, wobei der Schwerpunkt auf der Ein-/Ausgabe, dem Dateisystem und der Speicherzuweisung liegt.

Es enthält auch ein Referenzhandbuch für die Sprache. Die offizielle Erklärung der Syntax und Semantik von C ist der ANSI-Standard.

**Reply by ghost** [reply]
```#include<stdio.h>

int main(void)
{
  int c;

  while((c=getchar())!=EOF)
  {
    if(c=='\t')
    {
      putchar('\\');
      putchar('t');
    }

    if(c=='\b')
    {
      putchar('\\');
      putchar('b');
    }

    if(c=='\\')
    {
      putchar('\\');
      putchar('\\');
    }

    if(c !='\t' && c !='\b' && c!= '\\')
      putchar(c);
  }

return 0;
}
```

**Comment by madasy** [comment]
### B:

1. Bääbi: Puppe: Ich habe eine Puppe. 
2. baaie: baden: Ich gehe in die Limmat baden. 
3. bachab: zunichte, verloren: Es gaht alles bachab. 
4. Badi: Badeanstalt: Ich gang it Badi. 
5. Bäredräck: Lakritze: Ich hasse Bäredräck. 

![image](https://user-images.githubusercontent.com/41067156/200360928-4dd849a6-ce32-4cd5-9eac-8bb65f149756.png)
6. bcho: zurechtkommen, erreichen: Ich chan alles bcho wo ich chan wele. (Ich habe alles erreicht, wo ich wollte.) 
7. begäre: wünschen: Ich begäre erfolgrich si. 
8. Beiz: Wirtshaus: Gömmer hüt abig in e Beitz?

**Comment by esthicodes** [comment]
### C: 

- [ ] Cervelat: Fleischwurst
![image](https://user-images.githubusercontent.com/78131082/200393384-f33b9898-1a8e-4bbd-9ad9-6cf31fa77f4b.jpeg)
> Cervelat: Ech esse eh feini Cervelat (Olten) 🌭

- [ ] chääre: quengeln, un etwas bitten

![image](https://user-images.githubusercontent.com/78131082/200393712-514f6603-7ac2-4513-95f4-b4bd0aa32ed3.jpeg)

> chääre: D'Chender chääre weder Mou.
Die Kinder streiten wieder mal. 👧🧒👦👶

![image](https://user-images.githubusercontent.com/78131082/200393985-8ac9732c-123e-4ac0-87c4-7f07f697b0f2.jpeg)


- [ ] Chabis: Weisskohl, Kohlkopf, 1. Kopf, 2.Stumpfsinn, Unwahrheit

> Mini Schwöschder esst Chabis zom z'Mettag. 👧👧

![image](https://user-images.githubusercontent.com/78131082/200393977-07245808-0592-4ce2-b73f-bd144c3b52e0.jpeg)

**Reply by esthicodes** [reply]
- [ ] Chachle: flache Schlüssel 🔐 
- [ ] chäch: gesund, rüstig 
- [ ] Chänel: Dachrinne

**Reply by ghost** [reply]
Schüssel* 🍚🥗

**Comment by andisichert** [comment]
Morgens gehe ich dreißig Minuten schwimmen

**Reply by andisichert** [reply]
Zum Gitarre spielen brauch man flinke Finger
